launchable/commands/record/tests.py

```diff
@@ -80,19 +81,39 @@
 @click.pass_context
 def tests(context, base_path: str, session: Optional[str], build_name: Optional[str], post_chunk: int, subsetting_id: str,
           flavor, no_base_path_inference):
+    org, workspace = get_org_workspace()
+
+    if org is None or workspace is None:
+        raise click.UsageError(click.style(
+            "Could not identify Launchable organization/workspace. "
+            "Please confirm if you set LAUNCHABLE_TOKEN or LAUNCHABLE_ORGANIZATION and LAUNCHABLE_WORKSPACE environment variables",
+            fg="red"))
+
+    client = LaunchableClient(test_runner=context.invoked_subcommand,
+                              dry_run=context.obj.dry_run)
+    res = client.request("get", "verification")
+    if res.status_code == HTTPStatus.UNAUTHORIZED:
+        raise click.UsageError(click.style("Authentication failed. Most likely the value for the LAUNCHABLE_TOKEN "
```

Comment:
There is also a tokenless mode, so you should change the error message.
https://docs.launchableinc.com/getting-started/using-the-cli-with-a-public-repository

Reply:
fixed
https://github.com/launchableinc/cli/pull/360/commits/b00851afcb13d27e8626430e6c5f8a019426ef2a

Reply:
removed requesting verification after discussion with konboi-san for it is too much to do the verification here.
https://github.com/launchableinc/cli/pull/360/commits/b10d4071ee7d78aa8f3c2ba580316a48203391c2